README.md

```diff
@@ -161,6 +161,29 @@ the "generic" system, a new empty Spack environment will be automatically create
 different Spack environment by setting the environment variable `EXCALIBUR_SPACK_ENV`
 described above.
 
+### Local usage
+
+You can use the framework to run tests on a local system without a scheduler. After spack has
+been installed following the instructions given above, it needs to be told about the software
+installed on the system. On supported HPC systems this is handled by the spack environment. By default an
+empty spack environment will be created in `spack_environments/generic/default` when you run
```

Comment:
```suggestion
empty spack environment will be created in `spack-environments/generic/default` when you run
```
Side note, we should probably shorten the name of the `spack-environments` directory.